From 82dbd5c7f4c98dd645802483ea771b87a324e6fa Mon Sep 17 00:00:00 2001
From: Shaohui Xie <Shaohui.Xie@nxp.com>
Date: Tue, 19 Apr 2016 16:18:51 +0800
Subject: [PATCH 137/245] config: freescale: enable several PHY drivers

commit 5afde67f02fa08096e32c89701c6c0b87d1e9f78
[context adjustment]

Some PHYs can be found on LAYERSCAPE platforms, this commit enables
drivers for Aquantia, Realtek, Vitesse and fixed PHYs.

Signed-off-by: Shaohui Xie <Shaohui.Xie@nxp.com>
Integrated-by: Zhao Qiang <qiang.zhao@nxp.com>
---
 arch/arm64/configs/freescale.config | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm64/configs/freescale.config b/arch/arm64/configs/freescale.config
index 84b9d65..576841b 100644
--- a/arch/arm64/configs/freescale.config
+++ b/arch/arm64/configs/freescale.config
@@ -12,6 +12,11 @@ CONFIG_IOMMU_SUPPORT=y
 CONFIG_ARM_SMMU=y
 # mdio
 CONFIG_FSL_XGMAC_MDIO=y
+# phy
+CONFIG_AQUANTIA_PHY=y
+CONFIG_VITESSE_PHY=y
+CONFIG_REALTEK_PHY=y
+CONFIG_FIXED_PHY=y
 # dpaa2
 CONFIG_STAGING=y
 CONFIG_FSL_MC_BUS=y
-- 
2.1.0.27.g96db324

